src/network/netdev/bridge.c

```diff
@@ -133,6 +133,12 @@ static int netdev_bridge_post_create(NetDev *netdev, Link *link, sd_netlink_mess
                         return log_netdev_error_errno(netdev, r, "Could not append IFLA_BR_STP_STATE attribute: %m");
         }
 
+        if (IN_SET(b->igmp_version, 2,3)) {
```

Comment:
Maybe, we should introduce `bridge_verify()` to check the value.

Reply:
yes that would be better in the next version.  good idea.

Reply:
i'd probably add a specific parser just for this

Reply:
OK that would also do thanks.